spec/system/provider_interface/provider_changes_a_course_before_point_of_offer_spec.rb

```diff
@@ -215,6 +219,12 @@ def and_i_select_a_course_with_one_study_mode_and_one_location
     choose @one_mode_and_location_course.name_and_code
   end
 
+  def when_i_click_back_i_go_to_the_course_page
+    click_on 'Back'
+    expect(page).to have_content "Update course - #{application_form.full_name}"
+    expect(page).to have_content 'Course'
+  end
```

Comment:
Could split this into two separate methods, one's an action and the other's an expectation.

Reply:
Yeah fair point – I mean, this method already exists in `then_i_see_a_list_of_courses_to_select_from`. I'm not 100% sure on how i feel about the testing approach here – feel it needs to be a bit more in depth in general but also don't want to inflate the spec too much 🤔

Reply:
Yeah the back link is mainly based on that `wizard.previous_step` which is already tested somewhere else. I think it's probably the right amount of depth to check that it works as implemented, and not every single page.